source/manual/merge-pr.html.md

```diff
@@ -64,6 +66,14 @@ git push --set-upstream origin thomasjhughes-patch-1
 
 Change `thomasjhughes` to the GitHub username of the contributor and `patch-1` to the name of the branch in their fork.
 
+#### Dependabot
+
+Dependabot raises PRs whenever it sees new versions of gems available that are required by our applications.
+
+Dependabot is an external contributor and is therefore be subject to the same due diligence checks set out above as any other external contributor would have to go through, requiring two people from GDS to approve the PR before it can be merged. Particular attention should be paid to the changelog(s) of the upgraded gem(s) to ensure that there no unintended side-effects are introduced.
```

Comment:
"that there no unintended" - should be "that no unintended"

Reply:
Also, "is therefore be subject" should be "is therefore subject"

Reply:
Thanks, I don't know what happened to me here!